P5/Source/Specs/msName.xml

```diff
@@ -19,6 +18,7 @@ nominum</soCalled>, or nickname.</desc>
   <desc versionDate="2007-05-04" xml:lang="es">contiene cualquier forma de nombre alternativo no estructurado usado para un manuscrito, como por ejemplo <soCalled>ocellus nominum</soCalled>, o sobranombre.</desc>
   <desc versionDate="2007-01-21" xml:lang="it">contiene un qualsiasi nome alternativo non strutturato utilizzato per un manoscritto, per esempio <soCalled>ocellus
 nominum</soCalled>, o soprannome</desc>
+  <desc versionDate="2024-12-23" xml:lang="de">enthält einen Eigennamen in Form eines Nomens oder einer Nominalphrase, der für ein Manuskript oder anderes Objekt verwendet wird, anstatt einer formalen Identifikationsnummer oder Klassifikation.</desc>
```

Comment:
Disclaimer: I'm a German native speaker but not a librarian 

```suggestion
  <desc versionDate="2024-12-23" xml:lang="de">enthält einen Eigennamen in Form eines Nomens oder einer Nominalphrase, der für ein Manuskript oder anderes Objekt verwendet wird, anstatt einer formalen Identifikationsnummer oder Signatur.</desc>
```

I would replace "Klassifikation" (de) with "Signatur" (de) here for the English term "classmark" (en) is more or a less a synonym of "shelfmark" (en) which is usually translated as "Signatur" (de). "Klassifikation" (de) on the other hand is "classification (scheme)" (en).